web/src/Hangman.js

```diff
@@ -1,25 +1,54 @@
+import HangmanJSON from './contracts/Hangman.json';
 const ethers = require("ethers");
 
-class Hangman {
+export default class Hangman {
 
   constructor(HangmanFactoryContract) {
     this.Factory = HangmanFactoryContract;
     this.Game = null;
+    this.paymentAmount = 1;
```

Comment:
Can we have a TODO here to to mark this as constants? this value is only the Link cost of a chainlink transaction for Ropsten right? I'm totally going to forget

Reply:
yeah agreed. we have a lot of TODO sections.